docet-core/src/main/resources/META-INF/resources/docetres/docet/docet.js

```diff
@@ -227,7 +227,7 @@ var Docet = (function ($, document) {
         if (obj.offsetParent) {
             do {
                 curtop += obj.offsetTop;
-            } while (obj = obj.offsetParent);
+            } while (obj === obj.offsetParent);
```

Comment:
This could be breaking (It never worked I guess)

Reply:
What about changing the code in a way that the behavior is the same as before, but without this weird syntax

Reply:
I reproduced it.
That syntax was working because:
-  obj.offsetParent returns the closes HTMLDom element https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent
- In Javascript the variable assignment returns the assigned value:
```
var a = 1;
var b = null;
var result = (a = b);
```
Result will be null